torch/_dynamo/variables/lists.py

```diff
@@ -436,6 +437,16 @@ def call_method(
                 DequeVariable(list(items), regen_guards=False, **options),
             )
             return result
+        elif name == "appendleft" and self.mutable_local:
+            assert not kwargs
+            return tx.replace_all(
+                self,
+                DequeVariable(
+                    [args[0]] + list(self.items),
```

Comment:
You should do destructuring similar to as seen in extendleft so that you test that all arguments are actually used.

Reply:
Sorry, can you elaborate?